tests/acceptance/cli/lifecycle.py

```diff
@@ -90,46 +71,13 @@ def fetch_bob_public_keys(self) -> BobPublicKeys:
         return policy
 
 
-@pt.inlineCallbacks
-def test_federated_cli_lifecycle(click_runner,
-                                 testerchain,
-                                 random_policy_label,
-                                 federated_ursulas,
-                                 custom_filepath,
-                                 custom_filepath_2):
-    yield _cli_lifecycle(click_runner,
-                         testerchain,
-                         random_policy_label,
-                         federated_ursulas,
-                         custom_filepath,
-                         custom_filepath_2)
-
-
-@pt.inlineCallbacks
-def test_decentralized_cli_lifecycle(click_runner,
-                                     testerchain,
-                                     random_policy_label,
-                                     blockchain_ursulas,
-                                     custom_filepath,
-                                     custom_filepath_2,
-                                     agency_local_registry):
-
-    yield _cli_lifecycle(click_runner,
-                         testerchain,
-                         random_policy_label,
-                         blockchain_ursulas,
-                         custom_filepath,
-                         custom_filepath_2,
-                         agency_local_registry.filepath)
-
-
-def _cli_lifecycle(click_runner,
-                   testerchain,
-                   random_policy_label,
-                   ursulas,
-                   custom_filepath,
-                   custom_filepath_2,
-                   registry_filepath=None):
+def run_entire_cli_lifecycle(click_runner,
+                             testerchain,
```

Comment:
In relation with @derekpierre's comment below, it seems testerchain is only used in the decentralized test to get alice and bob accounts, so maybe you can get rid of it altogether.